.github/workflows/ui-test.yaml

```diff
@@ -19,9 +19,9 @@ jobs:
     - name: Install dependencies
       run: nix-env -if ui-tests/default.nix
 
-    - name: Start Vault
-      working-directory: on-premise-deploy/docker-compose/
-      run: . run-vault.sh
+    # - name: Start Vault
```

Comment:
You can just remove this since we've switched to the docker image. That and the `.sh` files.